decidim-comments/app/frontend/comments/comment.fragment.graphql

```diff
@@ -1,7 +1,12 @@
 fragment Comment on Comment {
-  body
-  createdAt
-  author {
-    name
+  ...CommentData
```

Comment:
Maybe we should add a comment (no pun intended) to make it clear that we have 3 levels of depth for a reason?